src/layers/IdentityCoordinatesTileLayer.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
This is mostly a copy and paste waiting for PR? Add a big comment at the top reminding us of that. And add more comments to flag the parts that you tweaked.

Reply:
Ok